src/vulnpy/common/utils.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+import vulnpy
+
+TEMPLATES_LOCATION = os.path.join(os.path.dirname(vulnpy.__file__), "templates")
+
+
+def get_template(path):
+    """
+    Read and return the contents of the file at TEMPLATES_LOCATON/<path>.
+    This is vulnerable to path traversal if used incorrectly, but security clearly
+    isn't a concern if you're using `vulnpy`.
```

Comment:
lol!

Reply:
I"m trying to think if there is a better way to access the template dynamically other than wit open, but I just can't think of one. This is clear enough that we can always iterate later on as we become more knowledgeable!

Reply:
I"m sure we could also later on optimize and instead of having each view call get_template and open the template, we could require users to run a make command and dump all templates into one place - reducing the load each view has to do. Later optimization for sure.

Reply:
This made me realize something interesting. We could load all of the templates to memory in vulnpy's top-level `__init__.py`, which will save us from re-reads.

We could even take advantage of that time to splice together the HTML fragments dynamically instead of relying on `make` and static files! We're basically designing the worlds most basic templating engine! I'm not going to make any changes here in this PR, but it's something I will be strongly considering in the near future.